src/AudioIO.h

```diff
@@ -345,6 +350,10 @@ class AUDACITY_DLL_API AudioIoCallback /* not final */
    RecordingSchedule mRecordingSchedule{};
    PlaybackSchedule mPlaybackSchedule;
 
+   struct TransportState;
+   //! Holds some state for duration of playback or recording
+   std::unique_ptr<TransportState> mpState;
```

Comment:
Could we rename this to mpTransportState? when I first encountered this member in the code, I got a bit confused because now "state" default-maps in my mind to RealtimeEffectState

Reply:
ok